Tests/SonarQube.Common.UnitTests/ConsoleLoggerTests.cs

```diff
@@ -186,41 +186,79 @@ public void CLogger_FormattedStrings()
                 ConsoleLogger logger = new ConsoleLogger(includeTimestamp: false);
 
-                logger.LogMessage("{ }");
+                logger.LogInfo("{ }");
                 output.AssertExpectedLastMessage("{ }");
 
-                logger.LogMessage("}{");
+                logger.LogInfo("}{");
                 output.AssertExpectedLastMessage("}{");
 
-                logger.LogMessage("a{1}2", null);
+                logger.LogInfo("a{1}2", null);
                 output.AssertExpectedLastMessage("a{1}2");
 
-                logger.LogMessage("{0}", "123");
+                logger.LogInfo("{0}", "123");
                 output.AssertExpectedLastMessage("123");
 
-                logger.LogMessage("a{0}{{{1}}}", "11", "22");
+                logger.LogInfo("a{0}{{{1}}}", "11", "22");
                 output.AssertExpectedLastMessage("a11{22}");
 
                 // 2. Logger with timestamps
                 logger = new ConsoleLogger(includeTimestamp: true);
 
-                logger.LogMessage("{ }");
+                logger.LogInfo("{ }");
                 output.AssertLastMessageEndsWith("{ }");
 
-                logger.LogMessage("}{");
+                logger.LogInfo("}{");
                 output.AssertLastMessageEndsWith("}{");
 
-                logger.LogMessage("a{1}2", null);
+                logger.LogInfo("a{1}2", null);
                 output.AssertLastMessageEndsWith("a{1}2");
 
-                logger.LogMessage("{0}", "123");
+                logger.LogInfo("{0}", "123");
                 output.AssertLastMessageEndsWith("123");
 
-                logger.LogMessage("a{0}{{{1}}}", "11", "22");
+                logger.LogInfo("a{0}{{{1}}}", "11", "22");
                 output.AssertLastMessageEndsWith("a11{22}");
             }
         }
 
+        [TestMethod]
+        public void CLogger_Verbosity()
+        {
+            ConsoleLogger logger = new ConsoleLogger(includeTimestamp: false);
+            Assert.AreEqual(logger.Verbosity, LoggerVerbosity.Info, "Default verbosity should be INFO");
+
+            using (OutputCaptureScope output = new OutputCaptureScope())
+            {
+                logger.Verbosity = LoggerVerbosity.Info;
+                logger.LogMessage(LoggerVerbosity.Info, "info1");
+                output.AssertExpectedLastMessage("info1");
+                logger.LogInfo("info2");
+                output.AssertExpectedLastMessage("info2");
+                logger.LogDebug("debug1");
+                output.AssertExpectedLastMessage("info2"); // the debug message should not have been logged
+
+            }
+
+            using (OutputCaptureScope output = new OutputCaptureScope())
+            {
+                logger.Verbosity = LoggerVerbosity.Debug;
+                logger.LogDebug("debug");
+                output.AssertExpectedLastMessage("debug");
+                logger.LogInfo("info");
+                output.AssertExpectedLastMessage("debug");
```

Comment:
This seems like an odd behaviour. Why wouldn't we always want info message to be output?